firestore-bigquery-export/extension.yaml

```diff
@@ -427,6 +428,28 @@ events:
     description:
       Occurs when the function is settled. Provides no customized data other
       than the context.
+
+  # NEW event types following the updated naming convention
+  - type: firebase.extensions.firestore-bigquery-export.v1.onStart
```

Comment:
we should using the extension ID instead of a hard coded name so multiple instances don't interfer

Reply:
I addressed in events.ts